CharacterSheet.API/DataAccess/D20Repo.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using ClassLibrary;
+using Microsoft.EntityFrameworkCore;
+
+namespace DataAccess
+{
+    class D20Repo : IRepo
```

Comment:
repository interface belongs in the domain model project (ClassLibrary), not in DataAccess, and should not have reference to anything in DataAccess project

Reply:
Will change.